fab_academy.html

```diff
@@ -407,14 +404,60 @@ <h5 class="mt-5 mb-2">Can I bring my own project to develop?</h5>
     font-size: 2rem;">
     Are you ready to change your future?</p>
-    <button type="button" 
-    class="btn btn-dark" 
-    style="width: 16rem;"       
-    data-toggle="modal" data-target="#applyModal">
+    <button type="button" class="btn btn-fablab-blue" style="width: 16rem;" data-toggle="modal" data-target="#applyModal">
     Request application
-    </button>  </div>
+    </button>
+  </div>
+
+</div>
+
 
+
+<!-- Sharing Buttons for Social Media -->
+<!-- https://gist.github.com/mskinna/6edcbd825aab2ba8fb5c -->
+
+<div class="d-flex flex-column box-btn-media">
+  <span>share</span>
+  <ul>
+
+    <li>
+      <!-- Sharingbutton Facebook -->
+      <a class="sm-link" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Ffablabbcn.org%2Ffab_academy.html&amp;src=sdkpreparse" target="_blank" rel="noopener" aria-label="Facebook">
+        <div class="sm-btn sm-btn-facebook">
+          <div aria-hidden="true" class="sm-btn-icon">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+            <path d="M18.77 7.46H14.5v-1.9c0-.9.6-1.1 1-1.1h3V.5h-4.33C10.24.5 9.5 3.44 9.5 5.32v2.15h-3v4h3v12h5v-12h3.85l.42-4z"/>
+          </svg>
+        </div>Facebook</div>
+      </a>
+    </li>
+
+    <li>
+      <!-- Sharingbutton Twitter -->
+      <a class="sm-link" href="https://twitter.com/intent/tweet?original_referer=https%3A%2F%2Ffablabbcn.org/fab_academy.html%2F&ref_src=twsrc%5Etfw&text=Did%20you%20know%20about%20Fab%20Academy%20Barcelona%3F%20Fab%20Academy%20is%20an%20intensive%20%EF%AC%81ve-month%20program%20that%20teaches%20students%20to%20envision%2C%20design%20and%20prototype%20projects%20using%20digital%20fabrication%20tools%20and%20machines." target="_blank" rel="noopener" aria-label="Twitter">
+        <div class="sm-btn sm-btn-twitter">
+          <div aria-hidden="true" class="sm-btn-icon">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+            <path d="M23.44 4.83c-.8.37-1.5.38-2.22.02.93-.56.98-.96 1.32-2.02-.88.52-1.86.9-2.9 1.1-.82-.88-2-1.43-3.3-1.43-2.5 0-4.55 2.04-4.55 4.54 0 .36.03.7.1 1.04-3.77-.2-7.12-2-9.36-4.75-.4.67-.6 1.45-.6 2.3 0 1.56.8 2.95 2 3.77-.74-.03-1.44-.23-2.05-.57v.06c0 2.2 1.56 4.03 3.64 4.44-.67.2-1.37.2-2.06.08.58 1.8 2.26 3.12 4.25 3.16C5.78 18.1 3.37 18.74 1 18.46c2 1.3 4.4 2.04 6.97 2.04 8.35 0 12.92-6.92 12.92-12.93 0-.2 0-.4-.02-.6.9-.63 1.96-1.22 2.56-2.14z"/>
+          </svg>
+        </div>Twitter</div>
+      </a>
+    </li>
+
+    <li>
+      <!-- Sharingbutton E-Mail -->
+      <a class="sm-link" href="mailto:info@fablabbcn.org?subject=Did%20you%20know%20about%20Fab%20Academy%20Barcelona%3F&body=Fab%20Academy%20is%20an%20intensive%20%EF%AC%81ve-month%20program%20that%20teaches%20students%20to%20envision%2C%20design%20and%20prototype%20projects%20using%20digital%20fabrication%20tools%20and%20machines.%20Go%20to%20this%20link%3A%C2%A0https%3A%2F%2Ffablabbcn.org%2Ffab_academy.html" target="_self" rel="noopener" aria-label="E-Mail">
```

Comment:
`sm-link` sounds like **Small Link**
Can we call it `social-media-link` or similar, so it is less confusing?
Same goes for all of the `sm-` classes :nerd_face: